packages/terra-tabs/docs/README.md

```diff
@@ -0,0 +1,167 @@
+# Terra Tabs
+
+Tabs are containers used to organize content. They allow for quick switching between groups of contextually related content. Content is divided into different containers and each container is viewable one at a time. The user can switch between containers by selecting the corresponding tab control.
+
+Tabs are meant for organizing content that doesn’t need to be compared or accessed simultaneously or if the user needs quick access through a group of contextually related content. Tabs are not intended for navigation.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-tabs`
+
+## Usage
+
```

Comment:
It would be helpful to consumers to add some doc on controlled vs uncontrolled tabs similar to https://github.com/reactjs/react-tabs/#controlled-vs-uncontrolled-mode

They have a brief section on [uncontrolled tabs](https://github.com/reactjs/react-tabs/#uncontrolled-mode) and [controlled tabs](https://github.com/reactjs/react-tabs/#controlled-mode).

Reply:
:+1: